### PR TITLE
Carp crit\dead

### LINKED ISSUE #5485: Space Carp can go into critical, but it looks like they're dead
## Description

See carp.yml: Critical state is at 50 damage, then they die at 100 damage, but visually there's no distinction.

**Screenshots**
From Delete#2526:
image

### PR DESCRIPTION
**Pull request by FoLoKe**
## About the PR 

Fixes #5485 
If this okay I'm gonna adjust sprites for all other animals without crit or dead textures
(Yes - Holocarp has crit\dead sprite and texture)

**Screenshots**

image

**Changelog**

:cl:
- add: Added space carps "dead" sprite

### CODE REVIEW

**Comment by cheesePizza2** [comment]
why not flip dead one upside down? like how dead fish float belly up

**Comment by FoLoKe** [comment]
> why not flip dead one upside down? like how dead fish float belly up

They already flipped, there just was no "dead\crit" differences 
image
(first two are original for alive and dead)

File: Resources/Textures/Mobs/Aliens/Carps/space.rsi/meta.json [hunk 1]
```diff
@@ -23,6 +23,14 @@
         ]
       ]
     },
+    {
+      "name": "crit",
+      "delays": [
+        [
+          1
+        ]
+      ]
```
Comment (metalgearsloth) [inline Resources/Textures/Mobs/Aliens/Carps/space.rsi/meta.json:32]: Don't need delay

File: Resources/Textures/Mobs/Aliens/Carps/magic.rsi/meta.json [hunk 2]
```diff
@@ -23,6 +23,14 @@
         ]
       ]
     },
+    {
+      "name": "crit",
+      "delays": [
+        [
+          1
+        ]
+      ]
```
Comment (metalgearsloth) [inline Resources/Textures/Mobs/Aliens/Carps/magic.rsi/meta.json:32]: Don't need delay

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Pull request merged**